docs/source/data/resolvers.mdx

```diff
@@ -388,49 +374,75 @@ Resolver functions are passed four arguments: `parent`, `args`, `context`, and `
-| `context` | <p>An object shared across all resolvers that are executing for a particular operation. Use this to share per-operation state, including authentication information, dataloader instances, and anything else to track across resolvers. </p><p>See [The `context` argument](#the-context-argument) for more information.</p> |
-| `info` | <p>Contains information about the operation's execution state, including the field name, the path to the field from the root, and more. </p><p>Its core fields are listed in the [GraphQL.js source code](https://github.com/graphql/graphql-js/blob/main/src/type/definition.ts#L977-L986). Apollo Server extends it with a [`cacheControl` field](../performance/caching/#in-your-resolvers-dynamic).</p> |
+| Argument  | Description                                                                                                                                                                                                                                                                                                                                                                                                                |
+| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `parent`  | <p>The return value of the resolver for this field's parent (i.e., the previous resolver in the [resolver chain](#resolver-chains)).</p><p>For resolvers of top-level fields with no parent (such as fields of `Query`), this value is obtained from the `rootValue` function passed to [Apollo Server's constructor](/apollo-server/v3/api/apollo-server/#constructor).</p>                                               |
+| `args`    | <p>An object that contains all GraphQL arguments provided for this field.</p><p> For example, when executing `query{ user(id: "4") }`, the `args` object passed to the `user` resolver is `{ "id": "4" }`.</p>                                                                                                                                                                                                             |
+| `context` | <p>An object shared across all resolvers that are executing for a particular operation. Use this to share per-operation state, including authentication information, dataloader instances, and anything else to track across resolvers. </p><p>See [The `context` argument](#the-context-argument) for more information.</p>                                                                                               |
```

Comment:
Let's start calling this `contextValue`, which is the term used in `graphql-js` and is what we're now using wherever it's a field name on an object (vs `context` which is the function that produces it).  (In this case, it's just a positional argument so the name is something we make up in the docs.)

Reply:
We definitely could do this, but it'd be something we'd want to change **everywhere** (including Odyssey's courses) for consistency. 

I will say I do like the name `context` for this positional argument because it _does_ make you think about the `context` initialization function for your server. So that logic leap between "how does my server know this information" feels a little shorter. 

However, if you feel like `contextValue` is the better name and makes more sense, then we can do that!

Reply:
@trevor-scheer and @StephenBarlow just in case either of you have thoughts on this too.

Reply:
I agree it's a deceptively heavy lift of a change, but fortunately still not _tremendously_ heavy since the name of the parameter doesn't really ever matter since you can technically name it whatever you want.

I think I do lean more in favor of keeping `context`, just because it's a map of many properties and values, and `contextValue` might suggest something more "scalar"

Reply:
contextValue is the term we're going to be using in our API where there's actually a field, and it's also the term used by the graphql-js API.

If you think that contextValue is a bad name for the context value, then let's consider backing out the AS4 change where we change every other use of it from `context` to `contextValue`... it's helpful feedback! But if we're making the distinction "`context` is a function returning `contextValue`" we should make it everywhere!